### DISCUSSION TITLE
Margin added on print view of PDF

### DISCUSSION THREAD

**Discussion by jenb-jbcole** [dompdf/dompdf#3264] [Q&A]
Hello,

We are having this issue where generating the PDF looks ok on Chrome, but as soon as the user tries to print it, it adds margin left, right and bottom of paper. We are setting the default paper size to be A4, and this is also what the print view is using, so it's confusing why this is happening.
Additionally, I saw people suggesting `@page {margin:0px}` but this did not work. I'm not actually sure if there's another setup needed to get this `@page` syntax to work? All I know is that adding it to my css file made no difference (any other styles such as `@page {background-color:red}`  for example does not work either)

**Comment by bsweeney** [comment]
What are your print settings? If you "fit to page" I believe Chrome will fit the entire PDF (including margins) inside the boundaries of the printable area. You'll want to print the PDF "actual size" instead to have it fill the paper.

**Comment by jenb-jbcole** [comment]
![screenshot_borders](https://github.com/dompdf/dompdf/assets/142896411/38d4215f-da25-4955-85fd-5affed22ec11)
There doesn't seem to be a 'print actual size' option 🤔

**Reply by bsweeney** [reply]
In that dialog it should be under the "Scale" option. Though I believe an a4 PDF printed to a4 paper at 100 scale should fill the page. For example, I used the following and the print dialog shows that it would print as expected:

```
<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <style>
    @page {
      size: a4;
      margin: 0;
    }
    body {
      background-color: red;
    }
  </style>
</head>
<body>
</body>
</html>
```

Which version of Dompdf are you using? And are you sure the PDF is a4?

Also, I see your destination is set to "Save as PDF" but when I select that destination I don't see the "More settings" section so I am unable to fully test this scenario. What happens if you select an actual printer?
